PKGBUILD

```diff
@@ -25,6 +26,12 @@ sha256sums=('986125b92192e404a2f3af5db510d2d651c6301d218cbb66edd6013f8e8153b0'
 build() {
   cd "server-$pkgver"
   local build_date=$(date "+%F-%T" -d "@${SOURCE_DATE_EPOCH}")
+  (
+    cd ui
+    yarn
+    yarn build
```

Comment:
```console
yarn --non-interactive --frozen-lockfile
yarn --non-interactive --frozen-lockfile build
```
To avoid hangs due to prompt and catch unwanted lockfile changes.